### PR TITLE
info design #140 and licenses #147

### LINKED ISSUE #140: Update information section design and links
Edit by Luke:
The information screen needs updating to match the designs posted by @cjmlgrto below 👇 

Changes required
- [ ] WH Novel Coronavirus information button renamed to "WH Novel Coronavirus Website" and changed to purple
- [ ] Purple "WH Anaesthetic Microsite" button to be added, which links to https://whcovid19.wixsite.com/covid19
- [ ] Purple "ICU Microsite" button to be added, which links to https://whicu2020.wixsite.com/icuquickguide
- [ ] Lorium ipsum to be removed and replaced by "Built with ❤️ by a band of volunteers"

~Similar to #117 Add WH Novel Coronavirus website fixed so expertly by @yimingc2  😄 👍~

~So with the blessing of @Western-Health-Covid19-Collaboration/designers and @lukesleeman this one may be ready for dev?~

~Request:~
~**Add WH Anaesthesia COVID-19 to the information screen.**~
~The intention would be to directly link through the external browser~
~https://whcovid19.wixsite.com/covid19~

~This link should be the top one above WH Novel Coronavirus site~
~This is the WH anaesthesia COVID-19 microsite which @galtaforce has very nicely put together through over the last few weeks.~
~This mobile-friendly microsite contains a more comprehensive range of documents for our anaesthetic department, that runs in parallel with the app 😄.~
~The app is intended as a fast aid educational tool and cognitive aid (as per TestFlight description and onboarding screen).~

### LINKED ISSUE #147: Build a Licenses screen
We will probably need to add in a licenses screen for the various bits of open source software we are including

### LINKED ISSUE #174: Information - UI inconsistencies
- [x] A - Title is too large
- [x] B - Button height is taller than specified on Zeplin
- [x] C - Margin between buttons are bigger than in Zeplin
- [x] D - Font and color in About section doesn’t quite match
- [x] E - Icon should have rounded corners with more margin around it

Screenshot is on an iPhone 5s so that it’s easier to read in GitHub. Also checked against iPhone 11 Pro Max

Simulator Screen Shot - iPhone 5s - 2020-04-04 at 21 45 57

### PR DESCRIPTION
**Pull request by rockgecko-development**
-Update designs of info screen #140
-Design inconsistencies on info screen #174 
-Add license page #147

## Screenshots
image

image
image

### CODE REVIEW

File: lib/strings.dart [hunk 1]
```diff
@@ -37,4 +37,8 @@ This app is designed to be used by Anaesthetic and Critical Care doctors at West
   static const String ppeTitle = 'PPE Technique';
   static const String ppeDonningTitle = 'Donning Personal Protective Equipment';
   static const String ppeDoffingTitle = 'Doffing Personal Protective Equipment';
+
+  static const String licencesTitle = 'Third-Party Licenses';
+  static const String licencesMsg =
+      'WHAC19 is built using open source software';
```
Comment (maks) [inline lib/strings.dart:43]: :+1:

File: lib/view/info_view.dart [hunk 2]
```diff
@@ -75,41 +88,49 @@ class InfoView extends StatelessWidget {
...
-      ),
+      Center(
+          child: Container(
+              decoration: BoxDecoration(
+                  color: const Color.fromARGB(255, 12, 16, 62),
+                  borderRadius: BorderRadius.circular(6)),
+              padding: const EdgeInsets.all(4),
+              child: Image.asset(
+                'assets/images/icon/high_res_icon.png',
+                height: 50,
```
Comment (maks) [inline lib/view/info_view.dart:121]: usually all the dimens are multiples of 4, eg previously was 84 ?

File: lib/view/license_view.dart [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+
+import '../strings.dart';
+import '../style.dart';
+
+class LicenseView extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    // Using showLicensePage directly gives the wrong style for app bar, and
```
Comment (maks) [inline lib/view/license_view.dart:9]: @rockgecko-development is this a bug in the license widget we should report to Flutter SDK repo?

**Review by maks — COMMENTED** [review]
:tada: Just a couple of questions from me @rockgecko-development

File: lib/view/license_view.dart (hunk 3 above)
Comment (workerbee22) [reply lib/view/license_view.dart:9]: Yeah happy with this until we can investigate the showLicensePage issues. 👍

**Review by workerbee22 — APPROVED** [review]
LGTM

**Comment by workerbee22** [comment]
@rockgecko-development Perhaps just @maks 's size multiple issue left before we merge this?

File: lib/view/license_view.dart (hunk 3 above)
Comment (rockgecko-development) [reply lib/view/license_view.dart:9]: I think it's caused by the default theme that's configured. It was a green titlebar... I think after merging @workerbee22 's PR last night it became white text on white bg. Not sure if a bug in LicensePage, or a problem with how we have set the global app theme.
Either way, it's not what we want and I didn't see any other way around

File: lib/view/info_view.dart (hunk 2 above)
Comment (rockgecko-development) [reply lib/view/info_view.dart:121]: This is the size in the mock.
https://app.zeplin.io/project/5e7597a68651056664fb6e53/screen/5e75996c5f1e2a6dd0200cc1
image

**Comment by rockgecko-development** [comment]
I assume it doesn't matter given there's a new icon in the works anyway. If the new icon _includes_ rounded corners, the code can be simplified

**Pull request merged**